### PR TITLE
fix ingester stderr logger

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
This fixes the ingesters so that logs that go to stderr do NOT contain the RFC5424 error message, just the message and values.

Here is what happens now.

If you DO NOT redirect stderr:
1. logs come out to stderr as RFC5424
2. only error, critical, and fatal are emitted to stderr

If you DO redirect stderr:
1. logs going to the redirected stderr file are emitted as RFC5424
2. logs going to actual stderr (the journal) are emitted as raw with no timestamp (so the journal can handle it)
3. only error, critical, and fatal are emitted to stderr

This PR addresses #902 
This PR addresses #901

### CODE REVIEW

File: ingesters/winevents/main.go [hunk 1]
```diff
@@ -189,22 +188,26 @@ type levelLogger struct {
 }
 
 // levelLogger implements the log.LevelRelay interface
-func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg []byte) error {
+func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg, rawMsg string) error {
 	switch lvl {
 	case log.DEBUG:
 		if debugOn {
-			fmt.Fprintln(os.Stdout, string(msg))
+			fmt.Fprintln(os.Stdout, rawMsg)
```
Comment (david-fritz-gravwell) [inline ingesters/winevents/main.go:195]: Do we ever want log messages on stdout? Even debug should go to stderr right?

File: ingesters/winevents/main.go [hunk 2]
```diff
@@ -189,22 +188,26 @@ type levelLogger struct {
...
 	case log.WARN:
-		return l.elog.Warning(1, string(msg))
+		return l.elog.Warning(1, rawMsg)
 	case log.ERROR:
 		fallthrough
 	case log.CRITICAL:
 		fallthrough
 	case log.FATAL:
-		return l.elog.Error(1, string(msg))
+		return l.elog.Error(1, rawMsg)
```
Comment (david-fritz-gravwell) [inline ingesters/winevents/main.go:206]: Does the Error() method exit? Do we have a Fatal() method? It doesn't look like it from the fallthroughs above but I thought I'd ask.

**Review by david-fritz-gravwell — CHANGES_REQUESTED** [review]

File: ingesters/winevents/main.go (hunk 2 above)
Comment (kris-watts-gravwell) [reply ingesters/winevents/main.go:206]: the `l.elog.Error` ?

File: ingesters/winevents/main.go (hunk 1 above)
Comment (kris-watts-gravwell) [reply ingesters/winevents/main.go:195]: no, debug is only done when you are actively debugging with a specific flag.  ITs not part of the traditional log chain

File: ingesters/winevents/main.go (hunk 2 above)
Comment (kris-watts-gravwell) [reply ingesters/winevents/main.go:206]: yes, we have a fatal method, and it does exit.  This switch just controls what makes it "out", the caller then terminates

**Review by david-fritz-gravwell — APPROVED** [review]
LGTM

**Pull request merged**